js/src/data/selectors.js

```diff
@@ -120,6 +120,22 @@ export const getMCReviewRequest = ( state ) => {
 	return state.mc_review_request;
 };
 
+export const getAllowedCountries = ( state ) => {
+	return state.mc.policy_check.allowed_countries;
+};
+
+export const getIsSsl = ( state ) => {
```

Comment:
`getIsSsl` should match with the resolver name: `getIsStoreSsl` See more info here: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data/#resolvers

Reply:
Thanks so much for pointing this out. Updated.